velox/connectors/hive/storage_adapters/s3fs/tests/MinioServer.h

```diff
@@ -47,14 +47,22 @@ class MinioServer {
     return tempPath_->path;
   }
 
-  std::shared_ptr<const Config> hiveConfig() const {
-    const std::unordered_map<std::string, std::string> config({
+  typedef std::unordered_map<std::string, std::string> map_type;
+  std::shared_ptr<const Config> hiveConfig(
+      const map_type configUpdate = map_type()) const {
```

Comment:
nit: does {} not work here, e.g. can't you do

```
const std::unordered_map<std::string, std::string>& configUpdate = {}
```
nit: configUpdate -> configOverride

Reply:
Fixed both!